forge/src/runner.rs

```diff
@@ -480,13 +548,125 @@ impl<'a, DB: DatabaseRef + Send + Sync> ContractRunner<'a, DB> {
+        identified_contracts: &BTreeMap<Address, (String, Abi)>,
+    ) -> Result<Vec<TestResult>> {
+        let TestSetup { address, mut logs, mut traces, labeled_addresses, .. } = setup;
+
+        let start = Instant::now();
+        let prev_db = self.executor.db.clone();
+        let mut evm =
+            InvariantExecutor::new(&mut self.executor, runner, self.sender, identified_contracts);
+
+        if let Some(InvariantFuzzTestResult { invariants, cases }) =
+            evm.invariant_fuzz(funcs, address, self.contract, test_options.invariant_depth)
+        {
+            let results = invariants
+                .iter()
+                .map(|(k, test_error)| {
+                    let mut counterexample_sequence = vec![];
+
+                    if let Some(ref error) = test_error {
+                        // we want traces for a failed fuzz
+                        if let TestError::Fail(_reason, vec_addr_bytes) = &error.test_error {
+                            // Reset DB state
+                            self.executor.db = prev_db.clone();
+                            self.executor.set_tracing(true);
+
+                            for (sender, (addr, bytes)) in vec_addr_bytes.iter() {
+                                let call_result = self
+                                    .executor
+                                    .call_raw_committing(*sender, *addr, bytes.0.clone(), 0.into())
+                                    .expect("bad call to evm");
+
+                                logs.extend(call_result.logs);
+                                traces.push((TraceKind::Execution, call_result.traces.unwrap()));
+
+                                counterexample_sequence.push(CounterExample::create(
+                                    *sender,
+                                    *addr,
+                                    bytes,
+                                    identified_contracts,
+                                ));
+
+                                let error_call_result = self
+                                    .executor
+                                    .call_raw(
+                                        self.sender,
+                                        error.addr,
+                                        error.func.0.clone(),
+                                        0.into(),
+                                    )
+                                    .expect("bad call to evm");
+
+                                if error_call_result.reverted {
+                                    logs.extend(error_call_result.logs);
+                                    traces.push((
+                                        TraceKind::Execution,
+                                        error_call_result.traces.unwrap(),
+                                    ));
+                                    break
+                                } else {
+                                    // todo should we show
```

Comment:
I lean towards no but argument could definitely be made to show calls to invariants that pass. I think we should *not* include them right now based on my previous experience showing them and it being a bit confusing/crowded